packages/remix-dev/cli/create.ts

```diff
@@ -205,12 +205,14 @@ export async function createApp({
   if (installDeps) {
     let packageManager = getPreferredPackageManager();
 
-    let npmConfig = execSync(
-      `${packageManager} config get @remix-run:registry`,
-      {
+    let npmConfig;
+    try {
+      npmConfig = execSync(`${packageManager} config get @remix-run:registry`, {
         encoding: "utf8",
-      }
-    );
+      });
+    } catch (error: unknown) {
```

Comment:
Since we're not using `error` at all, we can remove it

```suggestion
    } catch {
```